worker/src/github.rs

```diff
@@ -60,8 +55,8 @@ pub fn build_gist_vc(pk: &JWK, github_username: String) -> Result<Credential> {
           {
               "sameAs": "http://schema.org/sameAs",
               "GithubVerification": "https://tzprofiles.com/GithubVerification",
-              "GithubVerificationMessage": {
-                  "@id": "https://tzprofiles.com/GithubVerificationMessage",
+              "GithubAttestation": {
```

Comment:
I think `Attestation` is less clear than `GithubVerificationMessage`--it's vague and could refer to a Verifiable Credential, Proof, or on-chain claim. Perhaps `GitHubProofMessage`? Also, `GitHub` or `Github`? I think the former is more aligned with their brand guidelines?

Reply:
Updated references to be "GItHub" generally and "GitHubVerificationMessage" for consistency with other TZP VCs